scripts/recipe-template.txt

```diff
@@ -1,9 +1,17 @@
 cask "action-server" do
```

Comment:
Need to change the file to Action Server specific as we have RCC recipe inbound as well and they all live in this one repo and structure

Reply:
I'm not sure... we could just use the same recipe-template and just add one more argument to the `homebrew_update.py` (changing the template as needed).